pkg/cmd/pkg/package_install.go

```
@@ -1,26 +1,160 @@
 package pkg
 
 import (
+	"crypto/sha256"
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/dcos/client-go/dcos"
 	"github.com/dcos/dcos-cli/api"
+	"github.com/dcos/dcos-cli/pkg/plugin"
+	"github.com/dcos/dcos-cli/pkg/prompt"
+	"github.com/spf13/afero"
 	"github.com/spf13/cobra"
+
+	"github.com/dcos/dcos-core-cli/pkg/cosmos"
+	"github.com/dcos/dcos-core-cli/pkg/pluginutil"
 )
 
 func newCmdPackageInstall(ctx api.Context) *cobra.Command {
-	var appOnly, cliOnly, global, yes bool
+	var appOnly, cliOnly, yes bool
 	var appID, optionsPath, version string
 
 	cmd := &cobra.Command{
 		Use:   "install <package-name>",
 		Short: "Install a package",
 		Args:  cobra.ExactArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return invokePythonCLI(ctx)
+			packageName := args[0]
+
+			if cliOnly == appOnly {
+				cliOnly = true
+				appOnly = true
+			}
+
+			if optionsPath != "" {
+				_, err := os.Stat(optionsPath)
```

nit: maybe simpler not to have stat and let the read fail later on.

---

Then I'll need to change test that checks if we fail fast on wrong config